torch/_decomp/__init__.py

```diff
@@ -368,6 +368,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.tril_,
             aten.triu,
             aten.triu_,
+            aten.unbind,
```

Comment:
Inductor has lowering for this: implying inductor would want this op preserved in graph. 
Please exclude this from inductor decomp. 

see _inductor/lowering.py
and _inductor/decomposition.py

Reply:
I imagine inductor CI will break on this PR, right?